modules/cnf-assigning-a-secondary-network-to-a-vrf.adoc

```diff
@@ -0,0 +1,89 @@
+[id="cnf-assigning-a-secondary-network-to-a-vrf"]
+= Assigning a secondary network to a VRF
+
+[NOTE]
+====
+The feature described in this document is for Tech Preview purposes and is not supported by Red Hat at this time.
+====
+
+As a cluster administrator, you can configure an additional network for your VRF domain by using the VRF CNI plug-in. The virtual network created by this plug-in is associated with a physical interface that you specify.
+
+== Creating an additional network attachment with the VRF CNI plug-in
+
+The Cluster Network Operator (CNO) manages additional network definitions. When you specify an additional network to create, the CNO creates the NetworkAttachmentDefinition custom resource (CR) automatically.
+
+NOTE: Do not edit the NetworkAttachmentDefinition CRs that the Cluster Network Operator manages. Doing so might disrupt network traffic on your additional network.
+
+=== Prerequisites
+
+* Install the {product-title} CLI (oc).
+* Confirm that the VRF CNI is configured and deployed correctly. You can do this by ...
+* Log in to the OpenShift cluster as a user with cluster-admin privileges.
+
+=== Procedure
+
+. Edit the CNO CR by running the following command:
+
+[source,terminal]
+----
+$ oc edit networks.operator.openshift.io cluster
+----
+
+. Modify the CR that you are creating by adding the `rawCNIConfig` configuration for the additional network you are creating, as in the following example CR.
+.. `type` must be set to `vrf`.
```

Comment:
I would say here:
- plugins must be a list. The first item of the list must be the network configuration (for example, macvlan)
- the second item of the list is the vrf plugin configuration (and continue with the content below)